### DISCUSSION TITLE
HW 2 Questions

### DISCUSSION THREAD

**Discussion by UniquelyAbbi** [cooperrc/computational-mechanics#39] [General]
Hello, I started the home work, im using the book to help but seems I did something wrong.
![image](https://user-images.githubusercontent.com/106557922/172646218-f3062bda-ac00-4ee6-badd-ee5e87f82690.png)

**Comment by Aidan-Myatt** [comment]
You need to use parenthesis like this:

![image](https://user-images.githubusercontent.com/106316906/172652632-de50c794-4cff-4646-a3a4-a6ac09435b3a.png)

**Comment by cooperrc** [comment]
two notes:

1. you can include code in these comments with ` ``` ` e.g. 
````
```python
import pandas as pd
import numpy as np
!head ../data/transistor_data.csv
```
````

gives you

```python
import pandas as pd
import numpy as np
!head ../data/transistor_data.csv
```

2. if the file doesn't have spaces in the name, you don't _need_ the quotes `"-"`, but it will work with/without them. The error came up because there was one too many `.` ~`.`~`../data/` and `!head` needs a space after it
